cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
-comment: The metric enables to check visually e.g. MS1 TIC stability along the retention time and its absolute value compared to MS2 total ion chromatogram or other analyses MS1 TICs. TIC drops caused by e.g. spray stability issues or bubbles presence can be detected here.
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-is_a: MS:4000023 ! MS1 metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_relation MS:4000067 ! Total ion current chromatogram
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000109 ! Relative intensity
+name: number of identified spectra before FDR
+def: "Number of identified spectra the identification process provides." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000070
-name: MS2 Total ion current chromatogram
-def: "The total ion current chromatogram of MS2 measurements. The first column contains the retention time values in second, the second column the corresponding relative intensities." [PSI:QC]
-comment: The metric enables to check visually e.g. MS2 TIC absolute value compared to MS1 total ion chromatogram or other analyses MS2 TICs.
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-is_a: MS:4000024 ! MS2 metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_relation MS:4000067 ! Total ion current chromatogram
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000109 ! Relative intensity
+name: number of identified spectra after FDR
+def: "Number of identified spectra at given FDR threshold." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000071
-name: Fraction of peptide IDs around max MS1
-def: "Fraction of all peptides identified at least X minutes earlier or later than the most intense MS1 signal of the identification. The first column contains the time in minute-wise steps of seconds, the other comlumn the fraction of identified. Negative time values indicate minutes before and positive after maximum MS1." [PSI:QC]
-comment: Estimates very early peak broadening.
-is_a: MS:4000006 ! table
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000239 ! Fraction
+name: number of identified peptides
```

Comment:
To be consistand add "after FDR" to the name. The def describes it correctly.
Applies also to the following 5 terms.